lib/bundlex/toolchain/common/unix/os_deps.ex

```diff
@@ -0,0 +1,175 @@
+defmodule Bundlex.Toolchain.Common.Unix.OSDeps do
+  @moduledoc false
+
+  require Logger
+  alias Bundlex.Output
+
+  @precompiled_path "#{Mix.Project.build_path()}/bundlex_precompiled/"
```

Comment:
I'm not sure how this path can change, so I'd make it `defp` just in case

Reply:
done